awx/ui_next/src/components/Switch/Switch.jsx

```diff
@@ -0,0 +1,10 @@
+import { Switch } from '@patternfly/react-core';
+import styled from 'styled-components';
+
+Switch.displayName = 'PFSwitch';
+export default styled(Switch)`
+  display: flex;
+  flex-wrap: no-wrap;
+  /* workaround PF bug; used in calculating switch width: */
```

Comment:
Does PF know about this bug?

Reply:
https://github.com/patternfly/patternfly-react/issues/3281
This issue appears to be related